opencsp/common/lib/cv/spot_analysis/SpotAnalysisImagesStream.py

```diff
@@ -1,10 +1,13 @@
 from typing import Iterator
+from enum import Enum
+import functools
 
 from opencsp.common.lib.cv.CacheableImage import CacheableImage
 from opencsp.common.lib.cv.spot_analysis.ImagesIterable import ImagesIterable
 from opencsp.common.lib.cv.spot_analysis.ImagesStream import ImagesStream
 from opencsp.common.lib.cv.spot_analysis.ImageType import ImageType
 import opencsp.common.lib.tool.log_tools as lt
+import opencsp.common.lib.tool.typing_tools as tt
```

Comment:
Please remove these unused imports.

```suggestion
from typing import Iterator

from opencsp.common.lib.cv.CacheableImage import CacheableImage
from opencsp.common.lib.cv.spot_analysis.ImagesIterable import ImagesIterable
from opencsp.common.lib.cv.spot_analysis.ImagesStream import ImagesStream
from opencsp.common.lib.cv.spot_analysis.ImageType import ImageType
import opencsp.common.lib.tool.log_tools as lt
```